examples/ios/FLiOS/CoreMLClient/MLModelInspect.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import flwr
```

Comment:
I think the comment on line 14 can be removed  (I can't annotate it as from the previous version of the iOS example)

Reply:
removed.